samples/MiniZinc/data.dzn

```diff
@@ -0,0 +1 @@
+n=3;
```

Comment:
Are these two sample files from a real application or did you wrote them for this pull request? If from a real application, under what license were they distributed?